contrib/imgts/lib_entrypoint.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -e
+
+RED="\e[1;36;41m"
+YEL="\e[1;33;44m"
+NOR="\e[0m"
+SENTINEL="__unknown__"  # default set in dockerfile
+
+die() {
+    EXIT=$1
+    PFX=$2
+    shift 2
+    MSG="$@"
+    echo -e "${RED}${PFX}:${NOR} ${YEL}$MSG${NOR}"
+    [[ "$EXIT" -eq "0" ]] || exit "$EXIT"
+}
+
+# Pass in a list of one or more envariable names; exit non-zero with
+# helpful error message if any value is empty
+req_env_var() {
+    for i; do
+        if [[ -z "${!i}" ]]
+        then
+            die 1 FATAL entrypoint.sh requires \$$i to be non-empty.
+        elif [[ "${!i}" == "$SENTINEL" ]]
+        then
+            die 2 FATAL entrypoint.sh requires \$$i to be explicitly set.
+        fi
+    done
+}
+
+gcloud_init() {
+    set +xe
+    gcloud config set account "$GCPNAME" || die 3 FATAL setting account
+    gcloud config set project "$GCPPROJECT" || die 4 FATAL setting project
+    trap "rm -f /tmp/gcp.json" EXIT
+    echo "$GCPJSON" > /tmp/gcp.json && \
```

Comment:
Another great opportunity to use best practices such as `mktemp`

Reply:
Durrr, that's a great idea, thx!